docs/architecture3.md

```
@@ -0,0 +1,117 @@
+---
+
+## CardListPage
+
+### states
+
+- `cardsPromise` — `useState(getCardList)`로 생성, 재시도 시 새 Promise로 교체
+- `cardCount` — `CardList`에서 `onCountChange`로 올려받음
+
+### view
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)